en/ui-system/components/engine/multi-resolution.md

```diff
@@ -1,9 +1,10 @@
 # Auto fit for multi-resolution
 
-In the overall design of **Cocos Creator 3D**, it follows Cocos Creator's solution about adapting to screens with various resolutions with one set of assets. Generally speaking, we realize the adaptation scheme for multi-resolution through the following technology:
+**Cocos Creator 3D** still provides the solution about adapting to screens with various resolutions with one set of assets. Generally speaking, we realize the adaptation scheme for multi-resolution through the following technology:
```

Comment:
**Cocos Creator 3D** still provides the solution about adapting to screens with various resolutions with one set of assets.
-->
**Cocos Creator 3D** still provides the solution to adapting to screens with various resolutions with one set of assets.